packages/flutter/lib/src/animation/curves.dart

```diff
@@ -28,13 +28,31 @@ abstract class Curve {
   /// const constructors so that they can be used in const expressions.
   const Curve();
 
-  /// Returns the value of the curve at point `t`.
+  /// Returns the value of the curve at point [t].
```

Comment:
nit: this should be 

```suggestion
  /// Returns the value of the curve at point `t`.
```

Here and below since the `Curve` class does not have a member `t`. `t` is only a method argument.

Reply:
Fixed